calliope/test/test_example_models.py

```diff
@@ -76,8 +76,8 @@ def example_tester(self, solver="cbc", solver_io=None):
     def test_nationalscale_example_results_cbc(self):
         self.example_tester()
 
-    @pytest.importorskip("gurobipy")
     def test_nationalscale_example_results_gurobi(self):
+        pytest.importorskip("gurobipy")
```

Comment:
Why? gurobipy messing with things when you want to use the LP/file interface?

Reply:
It tries to import gurobipy, if it fails, it skips the whole test. This is for use on any machine (including the CI) where gurobi isn't available.